ui/pages/confirmations/confirmation/confirmation.js

```diff
@@ -12,9 +12,7 @@ import { isEqual } from 'lodash';
 import { produce } from 'immer';
 import log from 'loglevel';
 
-///: BEGIN:ONLY_INCLUDE_IF(snaps)
```

Comment:
Should we not keep this fencing for now?

Reply:
Mmmmmh yeah I don't know what happened, I did not intentionally removed these